workers/unity/Assets/Gdk/Core/MutableView/MutableView.cs

```diff
@@ -195,8 +197,34 @@ public bool TryGetEntity(long entityId, out Entity entity)
             return entityMapping.TryGetValue(entityId, out entity);
         }
 
-        public void AddGameObjectEntity(Entity entity, GameObject gameObject)
+        public bool TryGetEntityId(Entity entity, out long entityId)
         {
+            if (!entityManager.HasComponent<SpatialEntityId>(entity))
+            {
+                entityId = 0;
```

Comment:
Is this entirely necessary? Given the return value, its likely the `out` parameter  will be thrown away.

Reply:
This is because the compiler enforces that you assign out parameters a value. `entityId = something;` needs to be called at for all code paths at some point before the method returns.